controllers/packagebundlecontroller_controller.go

```diff
@@ -127,8 +126,7 @@ func (r *PackageBundleControllerReconciler) Reconcile(ctx context.Context, req c
 
 	err = r.bundleManager.ProcessBundleController(ctx, pbc)
 	if err != nil {
-		if strings.Contains(err.Error(), "connect: connection refused") &&
-			!r.webhookInitialized {
+		if !r.webhookInitialized {
```

Comment:
```suggestion
		if strings.Contains(err.Error(), "connect: connection refused") || !r.webhookInitialized {
```

Reply:
This is only looking for new bundles, so I think if we have a bundle an hour wait is fine to check for an update.